invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/RenameCommunitySlugButton.js

```diff
@@ -65,6 +65,8 @@ export class RenameCommunitySlugButton extends Component {
         this.setState({ error: invalidIdError });
       }
     }
+
+    this.setState({ loading: false });
```

Comment:
Agree, to be moved just after `if (error === "UNMOUNTED") return;`.
On success, it is not important because it will redirect.